### ISSUE TITLE
Error adding phone number. Steam returned "GeneralFailure"

### ISSUE THREAD

**Issue by dupatwoja** [Jessecar96/SteamDesktopAuthenticator#132]
I downloaded latest version of this program, I type login, password, give the email code, but then it asks me for phone number and anything I type it shows: 

Error adding phone number. Steam returned "GeneralFailure"
 
I had this phone number with mobile application and it worked fine, I also had it linked with Steam so I tried both with it linked and not. The same error. I tried the following formats:
 
+48123456789
+48123 456 789
+48123-456-789
+48 123456789
+48 123 456 789
+48 123-456-789
(+48 is my country code)
 
Everytime the same error. I also tried diffrent versions of the SDA, on older it just went back to main screen after giving phone number.

**Comment by hyt47** [comment]
are you missing a number?
+48 123-456-789X

my mobile phone number has 10 numbers
ex:
+48 072-222-2221

anyway:

it may be easier for u to add it using steam https://store.steampowered.com/account/

**Comment by yqp95470** [comment]
I had same issue. Steam blocks too frequent use of same phone number. Just wait a bit and try more

**Comment by vlogerM** [comment]
@hyt47 In Poland (+48) phone numbers has 9 digits.

**Comment by dupatwoja** [comment]
@hyt47, as @vLogerM said my phone is 9 digits and to the second part: "I also had it linked with Steam so I tried both with it linked and not. The same error."

@yqp95470 I tried after 7 days since last try this morning, the same error. I tried again just now, also the same thing. It has to be fault on program's end, not Steam as I can link the number perfectly fine without any errors in both Steam account settings and in android app. 

The same error appears when I type wrong login and password, so I guess what happens is that the SDA messes up my number and then Steam rejects it
